inferno-framework/davinci-pdex-test-kit
# FI-3378 Fix US Core importing

# Summary

There was a bug reported where we could not launch `bundle exec inferno console` with local US Core tests given the way they were imported in PDex. This PR fixes the bug.

# Testing Guidance

1. Checkout this branch
2. `bundle exec rspec`
3. Launch services and test kit and check that it works like main branch
4. Add the line `gem 'us_core_test_kit', path: '../us-core-test-kit'` to Gemfile and run `bundle exec inferno console` and confirm that it works
